Tweetinvi.Streams/AccountActivityStream.cs

```diff
@@ -228,18 +228,20 @@ private void TryRaiseUserEvent(string eventName, JObject jsonObjectEvent)
         private void TryRaiseMessageEvent(string eventName, JObject jsonObjectEvent)
         {
             var messageEventDTOs = jsonObjectEvent[eventName].ToObject<EventDTO[]>();
-            var apps = jsonObjectEvent["apps"].ToObject<Dictionary<string, App>>();
+            var apps = jsonObjectEvent.GetValue("apps")?.ToObject<Dictionary<string, App>>();
```

Comment:
Quick question, for my knowledge. Is there any case where the "apps" property would not exist on such an object?